Cython/Compiler/Symtab.py

```diff
@@ -540,7 +540,8 @@ def handle_already_declared_name(self, name, cname, type, pos, visibility):
                         # Note that we can override an inherited method with a compatible but not exactly equal signature, as in C++.
                         cpp_override_allowed = True
                     if cpp_override_allowed:
-                        entry = copy.copy(alt_entry)
+                        entry = (copy.copy if copy_entry else lambda e: e)(alt_entry)
```

Comment:
```suggestion
                        entry = alt_entry
                        if copy_entry:
                            entry = copy.copy(entry)
```

This feels like it's easier to read if written more directly.